### ISSUE TITLE
"LoadError [...] Evaluation into the closed module `GR`" from "Rerunning build step for GR package."

### ISSUE THREAD

**Issue by goretkin** [jheinen/GR.jl#394]
I am getting 

```julia
Your GR installation is incomplete. Rerunning build step for GR package.
ERROR: LoadError: LoadError: InitError: Evaluation into the closed module `GR` breaks incremental compilation because the side effects will not be permanent. This is likely due to some other module mutating `GR` with `eval` during precompilation - don't do this.
```

which I mitigated with `Pkg.build()`

but it suggests that this change:

https://github.com/jheinen/GR.jl/commit/3409914b5d4fb878ce4685f58ff1075f82e543cc#diff-7bd6c400cb5e7e66d9e2b4b11236ac0a15dea9293742c88f4cb31388602c2bcdL280-R297

is not behaving as intended. @mkitti 

```julia
julia> VERSION
v"1.6.0"
```

**Comment by goretkin** [comment]
For what it's worth, I would have preferred being just told to run `using Pkg; Pkg.build("GR")` manually in this case.

**Comment by goretkin** [comment]
Ah, I see. Yes, certainly a duplicate!

I had interpreted the change as, instead of instructing the user to _run_
the build step, it would inform the user that it was _running_.

**Comment by mkitti** [comment]
Duplicate and closed by #395

**Comment by mkitti** [comment]
Could someone close this issue please?
